tests/yast2_gui/yast2_hostnames.pm

```diff
@@ -44,12 +44,14 @@ sub run {
     assert_screen "yast2-$module-ui", 30;
     #	OK => Exit
     wait_screen_change { send_key "alt-o"; };
+    # Check that entry was correctly edited in /etc/hosts
+    select_console "root-console";
+    assert_script_run "grep '195\\.135\\.221\\.134\\sdownload\\.opensuse\\.org\\sdownload-srv' /etc/hosts";
```

Comment:
You can switch quotes or use `q` and `qq` to avoid escaping:
```perl
assert_script_run 'grep "195.135.221.134\sdownload.opensuse.org\sdownload-srv" /etc/hosts';
assert_script_run q#grep '195.135.221.134\sdownload.opensuse.org\sdownload-srv' /etc/hosts#;
```